src/frontend.rs

```diff
@@ -599,7 +601,12 @@ pub mod tests {
         let kyc = zu_kyc_pod_builder(&params, &gov_id, &pay_stub);
         println!("{}", kyc);
 
-        // TODO: prove kyc with MockProver and print it
+        let mut prover = MockProver {};
```

Comment:
This TODO was meant to mean to print the type `MainPod` which internally has been created with the MockProver, but that shouldn't matter in the frontend.
Something like this;
```rust
let mut prover = MockProver {};
let kyc_pod = kyc.prove(&mut prover).unwrap();
println!("{}", kyc_pod);
```

But the `Display` trait is not implemented for `frontend::MainPod` so that's what's actually missing to complete this.

Reply:
makes sense -- i've just reverted the file now since we don't have a clear picture of what the Display should show!